config_editor.html

```diff
@@ -24,9 +27,52 @@
     <meta name="theme-color" content="#ffffff">
 </head>
 <body>
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
+        <div class="navbar-brand"><img class="nav-logoimg" src="img/geyser.png"></div>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
+          aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a href="http://ci.geysermc.org">
+                        <button class="btn nav-link">
+                            <i class="fas fa-cloud-download-alt"></i>&nbsp;<span>Download</span>
+                        </button>
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="https://github.com/GeyserMC/Geyser">
+                        <i class="fab fa-github"></i>&nbsp;<span>Source</span>
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="http://discord.geysermc.org">
+                        <i class="fab fa-discord"></i>&nbsp;<span>Discord</span>
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="https://github.com/GeyserMC/Geyser/wiki/Setup">
+                        <i class="fas fa-tools"></i>&nbsp;<span>Setup</span>
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="config_editor.html">
+                        <i class="fas fa-sliders-h"></i>&nbsp;<span>Config Editor</span>
+                    </a>
+                </li>
+                <!--
+                <li class="nav-item">
+                    <a class="nav-link" href="https://patreon.com/GeyserMC"><i class="fab fa-patreon"></i> Donate</a>
+                </li>
+                -->
+            </ul>
+        </div>
```

Comment:
Could this be wrapped in a `<div class="container">` to pull the logo and buttons in so it doesn't look as strange with the rest of the page?

Reply:
Unfortunately doing so will throw off things inside the nav, and things start to move to the next line

Reply:
Gotta play around with it, but the other css we had was sort of wonky already so a lot of the changes I proposed were trying to fix up the weird styling.

Reply:
Resolved with invert when mobile